src/Phing/Io/FileSystem.php

```diff
@@ -869,14 +869,17 @@ public function compareMTimes($file1, $file2)
      */
     public function listContents(File $f)
     {
-        return array_keys(
+        $filenames = array_keys(
```

Comment:
No need for this temporary variable - just do it all in one :)

Reply:
Done :)